oem/ibm/libpldmresponder/file_io_by_type.cpp

```diff
@@ -576,5 +580,50 @@ int FileHandler::readFile(const std::string& filePath, uint32_t offset,
     return PLDM_ERROR;
 }
 
+int FileHandler::readFileByFd(int fd, uint32_t offset, uint32_t& length,
+                              Response& response)
+{
+    off_t fileSize = lseek(fd, 0, SEEK_END);
+    if (fileSize == -1)
+    {
+        error("file seek failed");
```

Comment:
Pls be specific on the seek failure as there is similar trace below for SEEK_SET. Mention something like lseek to the end failed

Reply:
done